CP05MOAS-PG583/CP05MOAS-PG583_R00004_ingest.csv

```diff
@@ -0,0 +1,8 @@
+parser,filename_mask,reference_designator,data_source,status,notes
+mi.dataset.driver.moas.gl.engineering.glider_eng_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-PG583/R00004/merged/cp_*.mrg,CP05MOAS-PG583-00-ENG000000,recovered_host,Available,
+mi.dataset.driver.moas.gl.ctdgv.ctdgv_m_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-PG583/R00004/merged/cp_*.mrg,CP05MOAS-PG583-01-CTDGVM000,recovered_host,Available,
+mi.dataset.driver.moas.gl.dosta.dosta_abcdjm_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-PG583/R00004/merged/cp_*.mrg,CP05MOAS-PG583-02-DOSTAM000,recovered_host,Available,
+mi.dataset.driver.moas.gl.flort_m.flort_m_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-PG583/R00004/merged/cp_*.mrg,CP05MOAS-PG583-03-FLORTM000,recovered_host,Available,
+mi.dataset.driver.moas.gl.flort_o.flort_o_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-PG583/R00004/merged/cp_*.mrg,CP05MOAS-PG583-04-FLORTO000,recovered_host,Available,
+mi.dataset.driver.nutnr_m.glider.nutnr_m_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-PG583/R00004/merged/cp_*.mrg,CP05MOAS-PG583-05-NUTNRM000,recovered_host,Available,
+mi.dataset.driver.moas.gl.parad.parad_m_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-PG583/R00004/merged/cp_*.mrg,CP05MOAS-PG583-06-PARADM000,recovered_host,Available,
```

Comment:
Looks good.  The NUTNR parser format is different than the others, but matches past ingestion CSVs.

Reply:
Looks good